README.md

```diff
@@ -1,10 +1,6 @@
 <p align="center">
   <a href="https://fingerprint.com">
-    <picture>
-     <source media="(prefers-color-scheme: dark)" srcset="https://fingerprintjs.github.io/home/resources/logo_light.svg" />
-     <source media="(prefers-color-scheme: light)" srcset="https://fingerprintjs.github.io/home/resources/logo_dark.svg" />
-     <img src="https://fingerprintjs.github.io/home/resources/logo_dark.svg" alt="Fingerprint logo" width="312px" />
-   </picture>
+    <img src="assets/logo_grey.svg" alt="Fingerprint logo" width="312px" />
```

Comment:
I have doubts if relative link will work for terraform

Reply:
Terraform registry handles some links differently, so we will just test and see how it goes. We can fix it if it doesn't work (we need the logo in `main` branch to fix it)